pkg/loki/modules.go

```diff
@@ -222,6 +223,26 @@ func (t *Loki) initQuerier() (services.Service, error) {
 	t.Cfg.Worker.MaxConcurrentRequests = t.Cfg.Querier.MaxConcurrent
 
 	var err error
+	if t.Cfg.Querier.PostFilterChunk {
+		enc, err := chunkenc.ParseEncoding(t.Cfg.Ingester.ChunkEncoding)
```

Comment:
`t.Cfg.Ingester.ChunkEncoding` refers to the current default encoding, which can be changed anytime. The same is the case for most values below that are being used from the config.

Reply:
done，👍 Thanks for your review and tips!